applications/esl/peripheral_esl/hw/esl_hw_impl.c

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/logging/log.h>
+
+#include "esl.h"
+
+LOG_MODULE_DECLARE(peripheral_esl);
+
+void hw_chrc_init(struct bt_esl_init_param *init_param)
+{
+	/** Initial Display elements here **/
+	for (uint8_t idx = 0; idx < CONFIG_BT_ESL_DISPLAY_MAX; idx++) {
+		init_param->displays[idx].width = CONFIG_ESL_DISPLAY_WIDTH;
+		init_param->displays[idx].height = CONFIG_ESL_DISPLAY_HEIGHT;
+		init_param->displays[idx].type = CONFIG_ESL_DISPLAY_TYPE;
```

Comment:
wouldn't it make more sense to have CONFIG_ESL_DISPLAY_0_WIDTH etc? Or use device tree for this?